src/utils/splitContentIntoElements.tsx

```diff
@@ -0,0 +1,30 @@
+interface Resources {
+  [key: string]: string;
+}
+
+export function splitContentIntoElements(
```

Comment:
We know that the utility is only used to replace link placeholders.

```suggestion
export function parseLinkPlaceholderContent(
```